docs/custom-component.md

```diff
@@ -1,22 +1,24 @@
 Custom Components
 ---------
 
-In order to provide the flexibility necessary to build an advanced search experience, Vue Instant Search includes the ability to build custom components.
+You can build you own components when the provided ones are not sufficient. Since we try
```

Comment:
Sorry if so, when reading the first sentence I felt like it was a lot of words to express what we wanted to say.

i.e. "in order to provide the flexibility necessary to build..."
and "includes the ability to build custom"

vs

You can build you own components when the provided ones are not sufficient.

I know this is boring writing but I guess that's just how I read/write it.